ui/src/app/pages/admin/admin-institution-edit.tsx

```diff
@@ -291,7 +292,11 @@ export const AdminInstitutionEdit = fp.flow(withNavigation, withRouter)(class ex
       institution: {
         shortName: '',
         displayName: '',
-        organizationTypeEnum: null
+        organizationTypeEnum: null,
+        tierConfigs: [{
+          ...defaultTierConfig(AccessTierShortNames.Registered),
+          membershipRequirement: null,  // the default is NOACCESS which also means "don't render the card"
```

Comment:
I don't quite follow the bug; we were starting with `membershipRequirement: NOACCESS`, and from this state, there was no way of  adding RT requirements?

Reply:
That's correct.

Edit mode is fine, since all Institutions have DOMAINS or ADDRESS for RT, and there's no way to transition _into_ NOACCESS.

Add mode and Edit mode are fine for CT, because NOACCESS is the norm and we have transitions in and out of NOACCESS - via an "enable Controlled Tier" switch.

Add mode for RT was just missed.